app/src/main/java/com/kamron/pogoiv/Pokefly.java

```diff
@@ -126,6 +123,13 @@
     private ImageView arcPointer;
     private LinearLayout infoLayout;
 
+    private LinearLayout touchView;
+    private WindowManager.LayoutParams touchViewParams;
+    private Handler screenScanHandler;
+    private Runnable screenScanRunnable;
```

Comment:
I'm choosing to keep it this way for the above reason. Let me know if you have a strong opinion/reason for changing it.